lab_3_keywords_textrank/main.py

```diff
@@ -51,7 +53,12 @@ def _clean_and_tokenize(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens
         """
-        pass
+        clean_text = ""
+        for symbol in text.lower():
+            if symbol not in self._punctuation:
+                clean_text += symbol
+        tokens = tuple(clean_text.split())
```

Comment:
do not create a variable. Return it immediately

Reply:
done